enginetest/queries/trigger_queries.go

```diff
@@ -2186,8 +2186,9 @@ INSERT INTO t0 (v1, v2) VALUES (i, s); END;`,
 				Expected: []sql.Row{{types.OkResult{}}},
 			},
 			{
-				Query:    "CALL add_entry(4, 'aaa');",
-				Expected: []sql.Row{{types.OkResult{RowsAffected: 1, InsertID: 3}}},
+				SkipResultCheckOnServerEngine: true, // call depends on stored procedure stmt for whether to use 'query' or 'exec' from go sql driver.
```

Comment:
Example:
`CREATE PROCEDURE p1() SELECT * FROM mytable;`
`CREATE PROCEDURE p2() INSERT INTO mytable VALUES (1), (2);`
Call on stored procedure `p1` need to be `.Query()` as it returns sql row results. (calling `.Exec()` on this procedure returns OkResult instead of the actual rows.)
Call on stored procedure `p2` need to be `.Exec()` as it returns `OkResult`. (calling `.Query()` on this procedure returns no result.)
The issue is that you can only call either `.Query` or `.Exec` on `CALL` statement depending on what the procedure statement is. This goes the same for `EXECUTE` statements as well, which depends what their prepared statements are.

Reply:
Ah... gotcha. Thanks for the explanation. I wonder if we could look at the expected result and use that as a hint for whether to call `Query` or `Exec`? We can tackle that in a separate PR, too, I'm just excited to brainstorm how we can close up more gaps over time 😄